plugin/k8s_ingress/README.md

```diff
@@ -0,0 +1,93 @@
+# k8s_ingress
+
+## Name
+
+*k8s_ingress* - resolves ingress FQDNs from outside Kubernetes clusters.
+
+## Description
+
+This plugin allows an additional zone to resolve the external IP address(es) of a Kubernetes
+Ingresses, similar to what `k8s_external` does for services. This plugin is only useful if the *kubernetes* plugin is also loaded.
+
+The plugin uses an external zone to resolve in-cluster IP addresses. It only handles queries for A,
+AAAA; all others result in NODATA responses. To make it a proper DNS zone, it handles SOA and NS queries for the apex of the zone.
```

Comment:
Even though it's implied, probably should call out specifically that PTR requests (reverse lookups) are not handled.